_includes/specials/taproot/ja/10-ln-with-taproot.md

```diff
@@ -0,0 +1,156 @@
+*LNプロトコル開発者[ZmnSCPxj][]の寄稿*
+
+この投稿では、[Taproot][topic taproot]がLNで可能にする2つのプライバシー機能について説明します:
+
+* LN上の[PTLC][topic ptlc]
+* P2TRチャネル
+
+### LN上のPTLC
+
+PTLCは[多くの機能][suredbits payment points]を可能にしますが、
+LNにとっての主な機能は、経路をランダム化することなく[支払いを非相関化][p4tr ptlcs]する機能です。[^route-randomization]
+シングルパスや[マルチパス][topic multipath payments]の経路に沿ったすべてのノードに、
+転送される各PTLCを調整するために使用されるスカラーを与えることができ、
+個々の転送が各LN支払い用の一意な識別子を[漏らす][news163 htlc problems]ことのない*支払いの非相関化*を可能にします。
+
+PTLCは***プライバシーの万能薬ではありません***。
+監視ノードが特定のタイムロックと量を持つ転送を発見し、
+その直後に2つめの監視ノードが*より小さな*タイムロックと*僅かに少ない*量を持つ転送を発見した場合、
+たとえ監視ノードが一意に識別可能なハッシュを介してそれらを相関させることができなくなったとしても、
+これらの転送が同じ支払いのパスに属する*可能性は非常に高い*です。しかし、得るものはあります:
+
+* 分析の不確実性が増します。監視者が扱うことのできる確率が低くなり、情報の価値ははるかに低くなります。
+* マルチパス支払いでは、より*多くの*非相関化が行われます。
+  個別のパスには、それぞれタイムロックや量に強い相関関係はないでしょうし、
+  LNが成功すれば、タイミングの相関関係も信頼できないほど十分な支払いがあるでしょう。
+* [HTLC][topic htlc]と比較してコストは増加しません
+  （[マルチシグの効率化][p4tr multisignatures]により、わずかにコストが減少する可能性もあります）。
+
+原則として、Taproot前のチャネルは、チャネルを閉じて再度開くことなく、
+PTLCをサポートするようアップグレードすることができます。
+既存のチャネルは、既存の非Taprootのファンディング・アウトプットを
+PTLCを含むTaprootアウトプットへ送信するオフチェーントランザクションを作成することで、PTLCをホストすることができます。
+つまり、LN上でPTLCのサポートを追加しても、
+各ノードとそのチャネルピアがソフトウェアをアップグレードする以上のコストがユーザーにかかることはないということです。
+
+しかし、PTLCを実際に使用するためには、送信者から受信者までの*すべての*転送ノードがPTLCをサポートする必要があります。
+つまり、十分な数のノードがアップグレードするまで、PTLCのサポートはほとんど使われない可能性があります。
+すべてのノードが同じプロトコルを使用する必要はありませんが（複数のPTLCプロトコルが存在する可能性があります）、
+すべてのノードが何らかのPTLCプロトコルをサポートする必要があります。
+複数のPTLCプロトコルをサポートすることになると、メンテナンスの負担が増えるため、
+そのようなプロトコルが多くならないことを*願っています*（理想的には１つだけ）。
+
+### P2TRチャネル
+
+ベースレイヤーとLNレイヤー間の非相関化を改善するための1つのソリューションは、
+LN上でその存在がゴシップされないチャネルである[非公開チャネル][topic unannounced channels]でした。
+
+残念ながら、すべてのLNチャネルは2人の署名者の協力を必要とし、現在のTaproot前のBitcoinでは、
+すべての2-of-2のScriptが*オープンに*コーディングされています。
+LNは2-of-2マルチシグの最も一般的なユーザーであるため、ブロックチェーンエクスプローラであれば、
+これが閉じられたLNチャネルであることを示すことができます。
+そこから資金を辿ることができ、それが別のP2WSHアウトプットに向かっていれば、
+それは*別の*非公開チャネルである可能性が高くなります。
+このように、非公開チャネルであっても、ある程度の誤検出はあるものの、閉じるとオンチェーンで特定することができます。
+
+Taprootは、[Schnorr署名][topic schnorr signatures]を使用することで、
+n-of-nを1-of-1と全く同じように見せることができます。
+少し工夫すれば、[k-of-n][topic threshold signature]でも1-of-1（およびn-of-n）と同じように見えます。
+そして、LNチャネルがP2TR UTXO、つまりP2TRチャネルを使うことで、
+非公開チャネルの*オンチェーン*プライバシーを向上させることができます。**[^two-to-tango]**
```

Comment:
I removed the bold formatting on this footnote.

Reply:
@bitschmidty thanks!